Description: Removed loading of local catalog.xml file because it is not
 suitable for Debian, changed the output format for html documentation
 because 'html' is not supported by current asciidoc package. Changed
 path for manpages because the manpage file is not present in the
 current upstream tarball.
Forwarded: not-needed
Author: Simone Rossetto <simros85@gmail.com>
Last-Update: 2018-11-25

--- a/doc/Makefile
+++ b/doc/Makefile
@@ -1,13 +1,13 @@
 .PHONY: all anchors outputs clean manpages
 
-export XML_CATALOG_FILES = catalog.xml
+#export XML_CATALOG_FILES = catalog.xml
 PY = python
 XSLTPROC_OPTS := --stringparam toc.section.depth 2 --stringparam generate.section.toc.level 1
 DBLATEX_OPTS  := -P latex.output.revhistory=0 -P doc.collab.show=1 -P toc.section.depth=2
 A2X_OPTS      := -a toc -a icons -L -d article -v --xsltproc-opts "$(XSLTPROC_OPTS)" --dblatex-opts "$(DBLATEX_OPTS)"
 
 %.html: %.txt anchors outputs
-	asciidoc -a data-uri -v -b html -d article -n -a toc2 -a icons $<
+	asciidoc -a data-uri -v -b html4 -d article -n -a toc2 -a icons $<
 
 # Ignore errors (dblatex may not be installed)
 %.pdf: %.txt anchors outputs
@@ -20,8 +20,8 @@ A2X_OPTS      := -a toc -a icons -L -d a
 all: guide.html guide.pdf guide.epub
 
 manpages:
-	cd man; ./create_manpage;
-	cd man; asciidoc -v -d manpage ./cxxtestgen.1.txt
+	cd ../debian/man; ./create_manpage;
+	cd ../debian/man; asciidoc -v -d manpage ./cxxtestgen.1.txt
 
 html: guide.html
 
